subprojects/docs/src/docs/userguide/commandLineInterface.adoc

```diff
@@ -0,0 +1,520 @@
+
+`--max-workers`::
+Sets maximum number of workers that Gradle may use. _Default is number of processors_.
+
+`--parallel`, `--no-parallel`::
+Build projects in parallel. For limitations of this option please see <<sec:parallel_execution>>. _Default is off_.
+
+`--profile`::
+Generates a high-level performance report in the `$buildDir/reports/profile` directory. `--scan` is preferred.
+
+`--scan`::
+Generate a build scan with detailed performance diagnostics.
+
+image:img/gradle-core-test-build-scan-performance.png[Build Scan performance report]
+
+==== Gradle Daemon
+You can manage the <<gradle_daemon,Gradle Daemon>> through the following command line options.
+
+`--daemon`, `--no-daemon`::
+Use the <<gradle_daemon, Gradle Daemon>> to run the build. Starts the daemon if not running or existing daemon busy. _Default is on_.
+
+`--foreground`::
+Starts the Gradle Daemon in a foreground process.
+
+`--status` (Standalone command)::
+Run `gradle --status` to list running and recently stopped Gradle daemons. Only displays daemons of the same Gradle version.
+
+`--stop` (Standalone command)::
+Run `gradle --stop` to stop all Gradle Daemons of the same version.
+
+`-Dorg.gradle.daemon.idletimeout=(number of milliseconds)`::
+Gradle Daemon will stop itself after this number of milliseconds of idle time. _Default is 10800000_ (3 hours).
+
+=== Environment Options
+You can customize many aspects about where build scripts, settings, caches, and so on through the options below. Learn more about customizing your <<build_environment, build environment>>.
+
+`-b`, `--build-file`::
+Specifies the build file. For example: `gradle --build-file=foo.gradle`
+
+`-c`, `--settings-file`::
+Specifies the settings file. For example: `gradle --settings-file=somewhere/else/settings.gradle`
+
+`-g`, `--gradle-user-home`::
+Specifies the Gradle user home directory. The default is the `.gradle` directory in the user's home directory.
+
+`-p`, `--project-dir`::
+Specifies the start directory for Gradle. Defaults to current directory. See <<sec:selecting_build>>.
+
+`--project-cache-dir`::
+Specifies the project-specific cache directory. Default value is `.gradle` in the root project directory.
+
+`-u`, `--no-search-upward` (deprecated)::
+Don't search in parent directories for a `settings.gradle` file.
+
+`-D`, `--system-prop`::
+Sets a system property of the JVM, for example `-Dmyprop=myvalue`. See <<sec:gradle_properties_and_system_properties>>.
+
+`-I`, `--init-script`::
+Specifies an initialization script. See <<init_scripts>>.
+
+`-P`, `--project-prop`::
+Sets a project property of the root project, for example `-Pmyprop=myvalue`. See <<sec:gradle_properties_and_system_properties>>.
+
+`-Dorg.gradle.jvmargs`::
+Set JVM arguments.
+
+`-Dorg.gradle.java.home`::
+Set JDK home dir.
+
+[[sec:cli_system_properties]]
+==== System Properties
+The following system properties are available for the `gradle` command.
```

Comment:
:o: Do we really only have one option here?

Reply:
Nope! I ported this and forgot we'd added ones for build cache and probably others.

Reply:
I don't actually see where the content on system properties went in your changes. Is this done in a separate PR?

Reply:
Yes, for the build environment chapter.